FueledUtils.xcworkspace/xcshareddata/FueledUtils.xcscmblueprint

```diff
@@ -14,7 +14,7 @@
     "46B19F3040765B0804498FD4689AE86ED8E3E630" : "FueledUtils\/",
```

Comment:
Oh, I had no idea it was not necessary. But yeah, looks redundant. I thought xcshareddata was only for things like shared schemes and shared breakpoints.